gradle/verification-metadata.xml

```diff
@@ -264,6 +274,17 @@
             <sha256 value="c2fd4366b1647233c99c70c17a2e90c89bf1ea60843749cca99d794525df9cd2" origin="Generated by Gradle"/>
          </artifact>
       </component>
+      <component group="com.fasterxml.jackson.core" name="jackson-annotations" version="2.13.3">
+         <artifact name="jackson-annotations-2.13.3.jar">
```

Comment:
we already have 2.13.4 of these jackson dependencies - is there a reason we need this specific version?

Reply:
One of the dependent libraries uses it.  Verification is based on the total closure of all possible dependency references, not on the set of actually used references.  You will also note that some SWT jars are also validated.